static/app/components/charts/simpleTableChart.tsx

```diff
@@ -76,7 +77,7 @@ function SimpleTableChart({
         getFieldRenderer(column.key, tableMeta);
 
       const unit = tableMeta.units?.[column.key];
-      let cell = fieldRenderer(row, {organization, location, eventView, unit});
```

Comment:
It is somewhat apparent that there is a design flaw somewhere given how we already pass location and organization. I think this likely stems from the way we've been using GridEditable (least by looking at the large set of changes being related to tables)